src/registrar/utility/csv_export.py

```diff
@@ -1077,6 +1088,29 @@ class DomainDataFull(DomainExport):
     Inherits from BaseExport -> DomainExport
     """
 
+    # NOTE - this override is temporary. Delete this after we consolidate these @property fields.
+    @classmethod
+    def get_fields(cls, model):
+        FIELDS = {
+            "Domain name": model.get("domain__name"),
+            "Status": model.get("status"),
+            "First ready on": model.get("first_ready_on"),
+            "Expiration date": model.get("expiration_date"),
+            "Domain type": model.get("domain_type"),
+            "Agency": model.get("federal_agency__agency"),
+            "Organization name": model.get("organization_name"),
```

Comment:
@zandercymatics so you're doing organization_name here instead of the suborganization field because of the error we are having on suborganization not being trimmed, is that correct? Asking due to the follow on ticket that will be touching this area (and should ideally change this to pull from suborg eventually)